incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/streams/OpenapiClientFactory.java

```diff
@@ -0,0 +1,1065 @@
+                    traceId, authorization, replyBudgetId, flag, reserved, payload, extension);
+
+            replySeq += reserved;
+        }
+
+        private void doOpenapiReplyFlush(
+            long traceId,
+            OctetsFW extension)
+        {
+            doFlush(sender, originId, routedId, replyId, replySeq, replyAck, replyMax,
+                traceId, authorization, replyBudgetId, 0, extension);
+        }
+
+        private void doHttpReplyEnd(
+            long traceId,
+            OctetsFW extension)
+        {
+            if (OpenapiState.replyOpening(state) && !OpenapiState.replyClosed(state))
+            {
+                doEnd(sender, originId, routedId, replyId, replySeq, replyAck, replyMax,
+                    traceId, authorization, extension);
+            }
+
+            state = OpenapiState.closeReply(state);
+        }
+
+        private void doHttpReplyAbort(
+            long traceId)
+        {
+            if (OpenapiState.replyOpening(state) && !OpenapiState.replyClosed(state))
+            {
+                doAbort(sender, originId, routedId, replyId, replySeq, replyAck, replyMax,
+                    traceId, authorization, EMPTY_OCTETS);
+            }
+
+            state = OpenapiState.closeInitial(state);
+        }
+
+        private void onOpenapiReplyReset(
+            ResetFW reset)
+        {
+            final long sequence = reset.sequence();
+            final long acknowledge = reset.acknowledge();
+            final int maximum = reset.maximum();
+            final long traceId = reset.traceId();
+
+            assert acknowledge <= sequence;
+            assert sequence <= replySeq;
+            assert acknowledge >= replyAck;
+            assert maximum >= replyMax;
+
+            replyAck = acknowledge;
+            replyMax = maximum;
+            state = OpenapiState.closeReply(state);
+
+            assert replyAck <= replySeq;
+
+            cleanup(traceId);
+        }
+
+        private void onOpenapiReplyWindow(
+            WindowFW window)
+        {
+            final long sequence = window.sequence();
+            final long acknowledge = window.acknowledge();
+            final int maximum = window.maximum();
+            final long traceId = window.traceId();
+            final long budgetId = window.budgetId();
+            final int padding = window.padding();
+            final int capabilities = window.capabilities();
+
+            assert acknowledge <= sequence;
+            assert sequence <= replySeq;
+            assert acknowledge >= replyAck;
+            assert maximum >= replyMax;
+
+            replyAck = acknowledge;
+            replyMax = maximum;
+            replyBud = budgetId;
+            replyPad = padding;
+            replyCap = capabilities;
+            state = OpenapiState.closingReply(state);
+
+            assert replyAck <= replySeq;
+
+            http.doHttpReplyWindow(traceId, acknowledge, budgetId, padding);
+        }
```

Comment:
Please group `onOpenapiXxx` methods together and `doOpenapiXxx` methods together.